meta-xt-control-domain/conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "xt-control-domain"
 BBFILE_PATTERN_xt-control-domain := "^${LAYERDIR}/"
 BBFILE_PRIORITY_xt-control-domain = "8"
 
-LAYERSERIES_COMPAT_xt-control-domain = "kirkstone"
+LAYERSERIES_COMPAT_xt-control-domain = "scarthgap"
```

Comment:
I believe we need both options. I.e. LAYERSERIES_COMPAT_xt-control-domain = "scarthgap kirkstone"